timbermill-java/timbermill-server/src/main/java/com/datorama/timbermill/server/service/TimbermillService.java

```diff
@@ -92,7 +98,10 @@ public TimbermillService(@Value("${INDEX_BULK_SIZE:200000}") Integer indexBulkSi
 							 @Value("${REDIS_POOL_MAX_IDLE:50}") int redisPoolMaxIdle,
 							 @Value("${REDIS_POOL_MAX_TOTAL:50}") int redisPoolMaxTotal,
 							 @Value("${REDIS_MAX_TRIED:3}") int redisMaxTries,
-							 @Value("${FETCH_BY_IDS_PARTITIONS:10000}") int fetchByIdsPartitions){
+							 @Value("${FETCH_BY_IDS_PARTITIONS:10000}") int fetchByIdsPartitions,
+                             @Value("${LIMIT_FOR_PERIOD:10000000}") int limitForPeriod,
```

Comment:
Too high let's put it at 30000 per minute

Reply:
done